# About
Scripts used for the article entitled "The transcriptomic profiling of SARS-CoV-2 compared to SARS, MERS, EBOV, and H1N1"
# Aim
The aim of this study was to perform a comparative analysis of transcriptomic signatures of infection of SARS-CoV-2 compared to other respiratory viruses (EBOV, H1N1, MERS-CoV, and SARS-CoV), to determine a unique anti-SARS-CoV-2 gene signature. 
# Results
We identified for the first time that molecular pathways for heparin-binding, RAGE, miRNA, and PLA2 inhibitors were associated with SARS-CoV-2 infection. The NRCAM and SAA2 genes, which are involved in severe inflammatory responses, and the FGF1 and FOXO1 genes, which are associated with immune regulation, were found to be associated with the cellular gene response to SARS-CoV-2 infection. Moreover, several cytokines, most significantly IL-8 and IL-6, demonstrated key associations with SARS-CoV-2 infection. Interestingly, the only response gene that was shared among the five viral infections was SERPINB1. The protein-protein interaction (PPI) analysis shed light on genes with high interaction activity that SARS-CoV-2 shares with other viral infections. The findings showed that the genetic pathways associated with rheumatoid arthritis, the AGE-RAGE signaling system, malaria, hepatitis B, and influenza A were of high significance. We found that the virogenomic transcriptome of infection, gene modulation of host antiviral responses, and GO terms of SARS-CoV-2 and EBOV were more similar than to SARS, H1N1, and MERS. This work compares the virogenomic signatures of highly pathogenic viruses and provides valid targets for potential therapy against SARS-CoV-2.

# Scripts structure :
These scripts are divded into two parts; 
 (A) Data used for this study.
 (B) scripts used for data filteration and manipulation.
 (C) Scripts used for results visualization. 
 (D) Articles' Results
 # Contacts of Authors
 hatem.zayed@qu.edu.qa and smahmoud@ageri.sci.eg
